### ISSUE TITLE
Add proxy settings

### ISSUE THREAD

**Issue by licaon-kter** [cloneconf/nedium#7]
like Tor (https://gitlab.com/guardianproject/NetCipher/) or I2P or etc.

**Comment by cloneconf** [comment]
Yes, its a cool feature when it's builtin especially tor, however current focus is on main functionality, still though this might be added in the future.

**Comment by cloneconf** [comment]
In the mean while, turn your vpn and you will be fine ;)
